src/com/jeopardy/Board.java

```diff
@@ -77,6 +79,37 @@ public String getAllQuestion() {
     return result.toString();
   }
 
+  public void processScore(boolean isCorrect, String playerName, int dollarValue){
+    //get contestant by name, loop through contestants list
+    Player currentPlayer = null;
+    for(Player player: getContestants()){
+      if(player.getName().equals(playerName)){
+        currentPlayer = player;
+        break;
+      }
+    }
+
+    //check isCorrect, increase for correct, decrease for incorrect
+    if(isCorrect){
+      currentPlayer.setScore(currentPlayer.getScore() + dollarValue);
```

Comment:
Player has addScore and deductScore methods.
You can change this code to 
currentPlayer.addScore(dollarValue)
and
currentPlayer.deductScore(dollarValue)